third-party/realdds/include/realdds/topics/readme.md

```diff
@@ -1,23 +1,24 @@
 
-The files in this directory are, for the most part, automatically generated from files in
-    `third-party/realdds/include/topics/*/*.idl`
-
-
-## Topic list
+# Topic List
 
 The various topics are arranged hierarchically:
 
 * `realsense/`
-    * `device-info` — device name, serial-number, product-line, lock status, and topic root enumeration
-    * `<model>_<serial-number>/` — topic root per device pointed-to in device-info
-        * `control` — client -> camera messaging
-        * `notification` — camera -> client notifications, responses, etc.
-* `rt/realsense/` — [ROS2](#ros2)-compatible streams
-    * `<model>_<serial-number>_<stream-name>` — Image/Imu stream supported by the device (e.g., Depth, Infrared, Color, Gyro, etc.)
+    * `device-info` — [device discovery](../../doc/discovery.md) ([flexible](flexible/))
+    * `<model>_<serial-number>/` — topic root per device in [device-info](../../doc/discovery.md) (all in [flexible](flexible/) format)
+        * `notification` — [server notifications](../../doc/notifications.md), responses, etc.
+        * `control` — [client requests to server](../../doc/control.md)
+        * `metadata` — [optional stream information](../../doc/metadata.md)
+* `rt/realsense/` — ROS2-compatible [streams](../../doc/streaming.md)
+    * `<model>_<serial-number>_<stream-name>` — [Image](https://github.com/ros2/common_interfaces/blob/rolling/sensor_msgs/msg/Image.msg)/[Imu](https://github.com/ros2/common_interfaces/blob/rolling/sensor_msgs/msg/Imu.msg) stream supported by the device (e.g., Depth, Infrared, Color, Gyro, etc.)
+
+# Interface Definition Files
+
+The code for each message type in this directory is generated from IDL files if the message format is defined by RealDDS.
 
-For a complete overview of the topics structure please refer to [DDS ICD](https://github.com/IntelRealSense/librealsense/blob/dds/third-party/realdds/doc/DDS%20ICD.md)
+For [ROS](#ros2), the IDLs are not available.
 
-## Generating
+# Generation
 
 A utility from FastDDS called [FastDDSGen](https://fast-dds.docs.eprosima.com/en/latest/fastddsgen/introduction/introduction.html#fastddsgen-intro) is required to convert the [IDL](https://fast-dds.docs.eprosima.com/en/latest/fastddsgen/dataTypes/dataTypes.html) files to the headers that're needed.
 Unfortunately, this utility requires a java installation and sometimes not even the latest.
```

Comment:
Last used version 2.7.1 or 2.9.1 as in the command?

Reply:
2.9.1. Fixed.